arrays/length.md

```diff
@@ -1,6 +1,6 @@
 # Length
 
-Arrays have a property called length, and it's pretty much exactly as it sounds, it's the length of the array.
+Arrays have a property called length, and it's pretty much exactly as it sounds. The length property is the length of the array.
```

Comment:
I do think is required/makes it easier to understand. The original line with the context is explanatory enough